src/components/AssetsListRow/AssetsListRow.js

```diff
@@ -98,6 +98,10 @@ export class AssetsListRow extends Component {
       return isPositive ? <GreenArrowUp /> : <RedArrowDown />;
     };
 
+    const currency = `${this.props.currency}`;
```

Comment:
why do you need back ticks?

Reply:
marketData?.price_change_24h_in_currency?.[currency]  when i'm using the [currency], I'm basically getting the data from the api with the naming of whatever value currency it's.   eg:  marketData?.price_change_24h_in_currency?.['usd']